app/components/file_attachment_list/file_attachment_image.js

```diff
@@ -95,14 +99,16 @@ export default class FileAttachmentImage extends PureComponent {
             imageSize,
             resizeMethod,
             resizeMode,
+            wrapperHeight,
+            wrapperWidth,
         } = this.props;
 
         let height = imageHeight;
         let width = imageWidth;
         let imageStyle = {height, width};
         if (imageSize === IMAGE_SIZE.Preview) {
-            height = 100;
-            width = this.calculateNeededWidth(file.height, file.width, height) || 100;
+            height = 80;
+            width = this.calculateNeededWidth(file.height, file.width, height) || 80;
```

Comment:
Nit: Do we need the || 80 here?  I see that you are returning the height from param which is 80